### PR TITLE
Updated __init__.py as for queue requirement

### PR DESCRIPTION
**Pull request by utkarsh0702**
Queue Implementation by Utkarsh

### CODE REVIEW

**Comment by czgdp1807** [comment]
In ArrayQueue use, DynamicOneDimensionalArray.
Refer, the discussions at #67

**Comment by utkarsh0702** [comment]
ok I will try

**Comment by utkarsh0702** [comment]
I need to ask something
I had wrote the code and committed it to but the number of lines or commit or project name is not appearing on my statistics page.

**Comment by czgdp1807** [comment]
From my experience, it takes around 24 hours for your contributions to reflect on the statistics page. However, feel free to contact the organizers if you are facing any issues.

**Comment by utkarsh0702** [comment]
I did the necessary changes for the Dynamic One Dimensional Array but it is still showing some errors.
Can you please help me with it.

**Comment by czgdp1807** [comment]
Resolve the conflicts and there are some indentation issues in your code. Once #69 is updated you will be able to see the coding pattern which you can apply in your branch.

**Comment by czgdp1807** [comment]
Please give a suitable title to your PR. `queue.py` isn't reflecting anything reasonable.

**Comment by utkarsh0702** [comment]
sir i updated the queue.py file according to the stack.py file

**Comment by utkarsh0702** [comment]
sorry sir i am new to this git PR concept so i am unable to change my PR name

File: pydatastructs/miscellaneous_data_structures/__init__.py [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 from . import (
     stack,
+
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/__init__.py]: Remove this line.

File: pydatastructs/miscellaneous_data_structures/__init__.py [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+    queue,
+
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/__init__.py]: Remove this line.

File: pydatastructs/miscellaneous_data_structures/__init__.py [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 from . import (
     stack,
+
+    queue,
+  
     binomial_trees
+
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/__init__.py]: Remove this line.

File: pydatastructs/miscellaneous_data_structures/queue.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
+from pydatastructs.linear_data_structures import DynamicOneDimensionalArray
+
+__all__ = [
+    'Queue'
+]
+
+_check_type = lambda a, t: isinstance(a, t)
+NoneType = type(None)
+rear= -1
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/queue.py]: Do not make rear a global variable.

File: pydatastructs/miscellaneous_data_structures/queue.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+class Queue(objects):
+    def __new__(clas, implementation='array', **kwargs):
+        if implementation == 'array':
+            return ArrayQueue(
+                kwargs.get('front', -1),
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/queue.py]: We can remove this from the API.

**Comment by czgdp1807** [comment]
It will be better, `rear` and `front` data members of `ArrayQueue` class and updating them on every `append` and `pop`.
eidt
Press the `Edit` button in your browser.

**Comment by utkarsh0702** [comment]
can you please check it

File: pydatastructs/miscellaneous_data_structures/__init__.py [hunk 6]
```diff
@@ -1,8 +1,8 @@
 __all__ = []
 
 from . import (
-    stack,
-    binomial_trees
```
Comment (czgdp1807) [inline pydatastructs/miscellaneous_data_structures/__init__.py]: This is needed.

**Comment by utkarsh0702** [comment]
can you please check it

**Comment by czgdp1807** [comment]
@utkarsh0702 Have you allowed edits from maintainers on this PR?

**Comment by utkarsh0702** [comment]
I am sorry I don't know how to do it

**Comment by czgdp1807** [comment]
Would you like to have your name included in the AUTHORS file? If yes then let me know your full name and email id that should be included in the file. Thanks.

**Comment by utkarsh0702** [comment]
Yes, i would love too.
Name: Utkarsh Mishra
Email Id: utkarsh07022000@gmail.com

**Pull request closed without merging**